app/helpers/application_helper.rb

```diff
@@ -165,17 +165,14 @@ def active_nav_li(link)
   end
 
   def show_time(length)
-    h = length / 60
-    min = length - h * 60
+    h, min = length.divmod(60)
```

Comment:
Shall we also ensure that length ain't nil/blank? 
`return unless length.present?`